Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/ReflectionAdapter.cpp

```diff
@@ -480,40 +536,51 @@ namespace AZ::DocumentPropertyEditor
-                        // it won't affect the calling DPE, however, other DPEs pointed at the same adapter would be unaware of the change,
-                        // and wouldn't update their UI.
-                        // In future, to properly support multiple DPEs on one adapter, we will need to solve this. One way would be to store
-                        // the json serialized value (which is mostly human-readable text) as an attribute, so any change to the Json would
-                        // trigger an update. This would have the advantage of allowing opaque and pointer types to be searchable by the
-                        // string-based Filter adapter. Without this, things like Vector3 will not have searchable values by text. These
-                        // advantages would have to be measured against the size changes in the DOM and the time taken to populate and parse them.
-                        return newValue;
-                    },
-                    false, hashValue);
+
+                if (IsInspectorOverrideManagementEnabled() && !serializedPathAttribute.GetString().empty())
+                {
+                    VisitValueWithSerializedPath(access, attributes);
+                }
+                else
+                {
+                    VisitValue(
+                        instancePointerValue,
+                        access.Get(),
+                        attributes,
+                        // this needs to write the value back into the reflected object via Json serialization
+                        [valuePointer = access.Get(), valueType = access.GetType(), this](const Dom::Value& newValue)
+                        {
+                            // marshal this new value into a pointer for use by the Json serializer
+                            auto marshalledPointer = AZ::Dom::Utils::TryMarshalValueToPointer(newValue, valueType);
+
+                            rapidjson::Document buffer;
+                            JsonSerializerSettings serializeSettings;
+                            JsonDeserializerSettings deserializeSettings;
+                            serializeSettings.m_serializeContext = m_serializeContext;
+                            deserializeSettings.m_serializeContext = m_serializeContext;
+
+                            // serialize the new value to Json, using the original valuePointer as a reference object to generate a minimal
+                            // diff
+                            JsonSerialization::Store(
+                                buffer, buffer.GetAllocator(), marshalledPointer, valuePointer, valueType, serializeSettings);
+
+                            // now deserialize that value into the original location
+                            JsonSerialization::Load(valuePointer, valueType, buffer, deserializeSettings);
+
+                            // NB: the returned value for serialized pointer values is instancePointerValue, but since this is passed by
```

Comment:
this is a good comment, but we should be tracking this as a GHI. Could you write one up? I think this is actually already testable, right? If you create an inspector looking at a prefab, then pin it, then create another Inspector looking at the same object, changes made in one of the Inspectors won't show up in the other one when there are pointers like this involved.

Reply:
Sure. Here it is https://github.com/o3de/o3de/issues/14583

This is a pre-existing comment. So I'm not 100% sure what it is suggesting. But if the solution is to store serialized values in DPE dom, this PR already does it behind a flag. So it might just be a case of testing out with an actual use case and seeing where things are at.

Few questions about your comment:

1. By 'pin it' , do you mean 'create an override' or something else?
2. What does create another inspector mean? There will only be 1 entity inspector in the editor right? I'm trying to understand what an actual use case for this would be. Please update it in the ghi created above and feel free to edit the issue if its missing any information.

Reply:
No, I mean pin it using the pin button:
![image](https://user-images.githubusercontent.com/85521892/218879360-2d1cf828-99ee-4718-a40d-a3e4b79d2634.png)
This leaves a "pinned" copy of the Inspector looking at the current Entity. There can be many inspectors open at the same time.